### DISCUSSION TITLE
Query Regarding Displaced Mesh in the New Moose Syntax System

### DISCUSSION THREAD

**Discussion by shrituntunroy** [idaholab/moose#28344] [Q&A Modules: Solid mechanics]
Hi Moose Team,

I’m a new Moose user and currently in the process of learning the platform. I have a question regarding the use of displaced meshes in the newer syntax system—specifically within the `Physics/SolidMechanics/QuasiStatic` system.

In the older syntax, this variable was specified like this:

```plaintext
[Kernels/TensorMechanics]
  use_displaced_mesh = true
[]
```

However, I couldn't find a corresponding input variable in the new `Physics/SolidMechanics/QuasiStatic` system. How can I specify the use of a displaced mesh in this updated syntax? Is it enabled by default, or is there another method to achieve this?

Thank you for your assistance.

**Comment by GiudGiud** [comment]
Hello

It should be turned on/off by default based on what models you are using, making the right selection for each option the Physics is giving you.
You should not need to worry about in the Physics.
In the simulation header, do you see the displacements variables being created?

Guillaume

**Reply by shrituntunroy** [reply]
Hi Giudicelli,

Thank you for your reply.

I am performing a crystal plasticity analysis.

Previously, I specified the displaced mesh option using the following settings:

```
[Kernels]
  [./TensorMechanics]
    strain = FINITE
    displacements = 'disp_x disp_y disp_z'
    use_displaced_mesh = true
  [../]
[]
```

You mentioned that the displaced mesh option will be enabled by default if I select the correct options in the physics section. Could you clarify which options I need to select?

Currently, I am using the following options:

```
[Physics]
  [SolidMechanics]
    [QuasiStatic]
      [all]
        strain = FINITE
        add_variables = true
        #use_displaced_mesh = true
      []
    []
  []
[]
```

Additionally, I'm having trouble understanding this part of your message:

```
In the simulation header, do you see the displacement variables being created?
```

Are you referring to the log file?

**Reply by GiudGiud** [reply]
I am referring to the log file

Yes the syntax should be basically the same between the old kernels syntax and the Physics

**Reply by shrituntunroy** [reply]
Thank you Giudicelli for your help.
